From 34dc2607db2052601886c55d4698b8158ef5306e Mon Sep 17 00:00:00 2001
From: Ng Zhi An <zhin@chromium.org>
Date: Wed, 12 May 2021 17:26:36 -0700
Subject: [PATCH] [wasm-simd][inspector] Use no_simd_sse to skip test

wasm-scope-info requires SIMD, since it prints the value of the SIMD
value in scope. We skip it using statusfile when SIMD is not supported.

Change-Id: Id64e130a1c497bae95ec5e794ad05816f8c908e7
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/2893568
Reviewed-by: Benedikt Meurer <bmeurer@chromium.org>
Commit-Queue: Zhi An Ng <zhin@chromium.org>
Cr-Commit-Position: refs/heads/master@{#74573}
---
 test/inspector/debugger/wasm-scope-info.js | 4 +---
 test/inspector/inspector.status            | 8 ++++----
 2 files changed, 5 insertions(+), 7 deletions(-)

diff --git a/test/inspector/debugger/wasm-scope-info.js b/test/inspector/debugger/wasm-scope-info.js
index 3ebac517e1..4b920d2b89 100644
--- a/test/inspector/debugger/wasm-scope-info.js
+++ b/test/inspector/debugger/wasm-scope-info.js
@@ -2,9 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// Flags: --experimental-wasm-type-reflection --experimental-wasm-simd
-// SIMD in Liftoff only works with these cpu features, force them on.
-// Flags: --enable-sse3 --enable-ssse3 --enable-sse4-1
+// Flags: --experimental-wasm-type-reflection
 
 utils.load('test/inspector/wasm-inspector-test.js');
 
diff --git a/test/inspector/inspector.status b/test/inspector/inspector.status
index 3cad9e676b..b342c9edfb 100644
--- a/test/inspector/inspector.status
+++ b/test/inspector/inspector.status
@@ -119,10 +119,10 @@
 }],  # 'arch == s390 or arch == s390x'
 
 ##############################################################################
-['(arch == mipsel or arch == mips64el) and not simd_mips', {
-  # Skip tests that fail on MIPS architectures that don't support SIMD.
-  'debugger/wasm-scope-info*': [SKIP],
-}],  # '(arch == mipsel or arch == mips64el) and not simd_mips'
+['no_simd_hardware', {
+  # Test contains SIMD value.
+  'debugger/wasm-scope-info': [SKIP],
+}], # no_simd_hardware
 
 ##############################################################################
 ['arch == riscv64', {
-- 
2.35.1

